src/components/organisms/SettingsRead.tsx

```diff
@@ -0,0 +1,248 @@
+  marginTop: 5,
+  color: mauve.mauve11,
+  fontSize: 12,
+  lineHeight: 1.5,
+});
+
+const Button = styled("button", {
+  all: "unset",
+  display: "inline-flex",
+  alignItems: "center",
+  justifyContent: "center",
+  borderRadius: 4,
+  padding: "0 15px",
+  fontSize: 15,
+  lineHeight: 1,
+  fontWeight: 500,
+  height: 35,
+
+  variants: {
+    variant: {
+      indigo: {
+        backgroundColor: indigo.indigo4,
+        color: indigo.indigo11,
+        "&:hover": { backgroundColor: indigo.indigo5 },
+        "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo7}` },
+      },
+    },
+  },
+
+  defaultVariants: {
+    variant: "indigo",
+  },
+});
+const Fieldset = styled("fieldset", {
+  all: "unset",
+  marginBottom: 15,
+  width: "100%",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "flex-start",
+});
+
+const Label = styled("label", {
+  fontSize: 13,
+  lineHeight: 1,
+  marginBottom: 10,
+  color: indigo.indigo12,
+  display: "block",
+});
+
+const Input = styled("input", {
+  all: "unset",
+  flex: "1 0 auto",
+  borderRadius: 4,
+  padding: "0 10px",
+  fontSize: 15,
+  lineHeight: 1,
+  color: indigo.indigo11,
+  boxShadow: `0 0 0 1px ${indigo.indigo7}`,
+  height: 35,
+  "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo8}` },
+  "&::placeholder": { color: mauve.mauve7 },
+});
+
+const FieldCheck = styled("fieldset", {
+  all: "unset",
+  width: "100%",
+  display: "flex",
+  alignItems: "center",
+});
+
+const CheckLink = styled("a", {
+  color: indigo.indigo11,
+  textDecoration: "underline",
+  "&:visited": { color: indigo.indigo11 },
+});
+
+type Props = {
+  setSettingsView: React.Dispatch<React.SetStateAction<string>>;
+  settingsView: React.Dispatch<React.SetStateAction<string>>;
+  selectedDeviceName: React.Dispatch<React.SetStateAction<string>>;
+  selectedDeviceIndex: React.Dispatch<React.SetStateAction<number>>;
+  deviceData: React.Dispatch<React.SetStateAction<[]>>;
+  userData: React.Dispatch<React.SetStateAction<{}>>;
+  isLoading: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+export default function SettingsRead({
+  setSettingsView,
+  settingsView,
+  selectedDeviceName,
+  selectedDeviceIndex,
+  deviceData,
+  userData,
+  isLoading,
+}: Props) {
+  // useEffect(() => {
+  //   let index: number = Number(selectedDeviceIndex);
+  //   console.log(deviceData[index]);
+  // }, []);
+
+  // console.log(settingsView);
+
+  // const setIndex = () => {
+  //   let index: number = Number(selectedDeviceIndex);
+  //   // let device: {} = dataArray[index];
+  //   console.log(deviceData[index]);
+  // };
+
+  // setIndex();
+
+  const openSettingsChange = () => {
+    setSettingsView("change settings");
+  };
+
+  if (isLoading === true) {
+    return <div className="loading">Loading...</div>;
+  } else {
+    return (
+      <>
+        <div className={styles.top}>
+          <div className={styles.top__inner}></div>
+          <div className="user-settings">
```

Comment:
Please change the className `"user-settings"` to `"styles.user-settings"`.
I recommend you to use underscore like `"user__settings"`.

Reply:
Done